### DISCUSSION TITLE
Upgrade from OKD 4.7 to 4.8

### DISCUSSION THREAD

**Discussion by dkoci** [okd-project/okd#1018] [Q&A]
Hi.
Is it possible to upgrade from the latest 4.7 (4.7.0-0.okd-2021-09-19-013247) version to 4.8? I dont see any updates available in my okd cluster.
Tnx

**Comment by vrutkovs** [comment]
Yes, see `Upgrades to` section on [release page](https://amd64.origin.releases.ci.openshift.org/releasestream/4-stable/release/4.7.0-0.okd-2021-09-19-013247)

**Reply by dkoci** [reply]
I dowloaded installer and client, but I still dont have upgrade to 4.8 option:

![image](https://user-images.githubusercontent.com/5565897/146004378-0ddb0ed2-588b-4a54-965d-b8849326890d.png)

**Reply by vrutkovs** [reply]
Wrong channel - it needs to be `stable-4`. See https://github.com/openshift/okd/discussions/788#discussioncomment-1050985 and other discussions here

**Reply by dkoci** [reply]
Successfully upgraded from 4.7.0-0.okd-2021-09-19-013247 to 4.8.0-0.okd-2021-10-01-221835.
Thank you
